doc/audit/builtin_maps.rst

```diff
@@ -122,7 +122,7 @@ Identity, status and attestations (endorsed quotes) of the nodes hosting the net
 ``nodes.code_ids``
 ~~~~~~~~~~~~~~~~~~
 
-Versions of the code allowed to join the current network.
+Versions of the code allowed to join the current network on :doc:`SGX <../operations/platforms/sgx>`, or measurement of the UVM on :doc:`SNP <../operations/platforms/snp>`.
```

Comment:
```suggestion
Versions of the code allowed to join the current network on :doc:`SGX <../operations/platforms/sgx>`, or measurement of the initial VM pages on :doc:`SNP <../operations/platforms/snp>`.
```

Reply:
Addressed both comments about what measurement means in https://github.com/microsoft/CCF/pull/4330/commits/5414843c99f4f9b082d5b470c4409c05422ff009